packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/model/NotificationContent.java

```diff
@@ -70,17 +84,39 @@ public Number getBadgeCount() {
     return mBadgeCount;
   }
 
-  public boolean shouldPlayDefaultSound() {
+  @Override
+  public boolean getShouldPlayDefaultSound() {
     return mShouldPlayDefaultSound;
   }
 
+  @Override
+  public boolean getShouldUseDefaultVibrationPattern() {
+    return mShouldUseDefaultVibrationPattern;
+  }
+
   @Nullable
-  public Uri getSound() {
-    return mSound;
+  public String getSoundName() {
+    return mSound != null ? mSound.getLastPathSegment() : null;
   }
 
-  public boolean shouldUseDefaultVibrationPattern() {
-    return mShouldUseDefaultVibrationPattern;
+  @Nullable
+  @Override
+  public Object getImage(@NonNull Context context, @NonNull Continuation<? super Bitmap> $completion) {
+    try {
+      ApplicationInfo ai = context.getPackageManager().getApplicationInfo(context.getPackageName(), PackageManager.GET_META_DATA);
+      if (ai.metaData.containsKey(META_DATA_LARGE_ICON_KEY)) {
+        int resourceId = ai.metaData.getInt(META_DATA_LARGE_ICON_KEY);
+        return BitmapFactory.decodeResource(context.getResources(), resourceId);
+      }
+    } catch (PackageManager.NameNotFoundException | ClassCastException e) {
+      Log.e("expo-notifications", "Could not have fetched large notification icon.");
```

Comment:
```suggestion
      Log.e("expo-notifications", "Could not have fetched large notification icon.", e);
```
Let's show the original error too.